lib/ossert/fetch/stackoverflow.rb

```diff
@@ -1,67 +1,165 @@
 # frozen_string_literal: true
 module Ossert
   module Fetch
-    # Class implementing `StackOverflow` crawler
+    # Class implementing `StackOverflow` crawler.
+    # An example of the typical response is listed in the end of the file.
     class StackOverflow
       attr_reader :client, :project
 
+      # @!attribute [r] client
+      # a trivial client for fetching web API
+      # @return [SimpleClient] instance of the prepared web page fetcher
+
+      # @!attribute [r] project
+      # project which will be filled during the processing step of the fetcher
+      # @return [Project] project filled by this fetcher
+
+      # for delegators
       extend Forwardable
+
+      # @!method community
+      # @see Project#community
+      # @return (see Project#community)
       def_delegators :project, :community
 
+      # Common URL for all StackExchange related queries
       BASE_URL = 'https://api.stackexchange.com/2.2/'
+
+      # A path for searching of questions
       SEARCH_URL = 'search/advanced'
+
+      # A template set of parameters for every query to StackExchange.
       SEARCH_PARAMS = {
         site: :stackoverflow,
         tagged: 'ruby',
-        filter: :total
+        pagesize: 100,
+        order: :asc,
+        sort: :creation
       }.freeze
 
-      # Public: Create StackOverflow fetcher
-      #
-      # project - an object representing a project
-      #           it should contain:
-      #             - #name method to return String;
-      #             - #community accessor to return
-      #                 and accept a Hash.
-      #
-      # Returns nothing.
+      # Maximum number of pages to be fetched for a single project
+      MAX_PAGES = 50
+
+      # Create StackOverflow fetcher
+      # @param project [Project] a project to fill
```

Comment:
It is `Ossert::Project` indeed. The other one probably should be specified with ::